src/core/simplify.rkt

```diff
@@ -150,6 +133,10 @@
          '((* x 6) (* 6 x))
          (simplify-expr '(+ (+ (+ (+ (+ x x) x) x) x) x) #:rules (*simplify-rules*)))
 
+  (check-equal?
+         '5/10
```

Comment:
Isn't `'5/10` the same as `1/2`? (No quote, fraction reduced?)

Reply:
Also---any reason this isn't part of the bigger block of simplifications we test above? Doing it in a batch helps the test run faster (not an issue now I guess) and implicitly tests bad interactions in the e-graph.